File: lite-flow-react/mock/flow.js
var Mock = require('mockjs');
var qs = require('qs');

let flowsData = Mock.mock({
    'data|20': [
        {
            'id|+1': 1,
            name: '@cname',
            description: '@cname',
            status: 1,
            user: {
                id: 1,
                userName: "lite"
            },
            createTime: 1540126236000,
            updateTime: 1540126236000
        }
    ]
});
module.exports = {
    'GET /console/flow/list': function (req, res) {
        const page = qs.parse(req.query);

        let pageSize = page.pageSize;
        let pageNum = page.pageNum;
        if (!pageSize) {
            pageSize = 10;
        }
        if (!pageNum) {
            pageNum = 1;
        }
        let data = flowsData.data;
        let result = data.length > pageSize ? data.slice((pageNum - 1) * pageSize, pageNum * pageSize) : data.slice();
        res.json({
            status: 0,
            data: result,
            total: data.length
        })
    },
    'POST /console/flow/addOrUpdate': {
        status: 0,
        data: 101
    },
    'POST /console/flow/delete': {
        status: 0
    }
};

File: lite-flow-console/lite-flow-console-model/src/main/java/cn/lite/flow/console/model/basic/Flow.java
package cn.lite.flow.console.model.basic;

import lombok.Data;
import lombok.ToString;

import java.io.Serializable;
import java.util.Date;

/**
 * 任务流
 */
@Data
@ToString
public class Flow implements Serializable {

    private Long id;

    private String name;                //任务名称

    private String description;         //任务描述

    private Long userId;                // 创建者id

    private Integer status;             // 任务流状态

    private Date createTime;            // 创建时间

    private Date updateTime;              //更新时间

}
